lib/parsers/entity_parser.js

```diff
@@ -376,10 +384,26 @@ function setDestinationAssociationsForClass(relatedRelationships, entityName) {
       relationship.otherEntityField = lowerFirst(splitField.otherEntityField);
       relationship.ownerSide = false;
     }
+    setOptionsOfRelationship(relatedRelationship, relationship);
     entities[entityName].addRelationship(relationship);
   });
 }
 
+function setOptionsOfRelationship(relatedRelationship, relationship) {
+  if (relatedRelationship.options) {
+    Object.entries(relatedRelationship.options).forEach(option => {
```

Comment:
same remark, you can do this `Object.entries(jdlField.options).forEach(([key, value]) => ...)`

Reply:
done